src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -430,8 +473,27 @@ internal void InvokeResponse(InvocationResponse invokeResponse)
             {
                 try
                 {
-                    IDictionary<string, object> bindingsDictionary = invokeResponse.OutputData
-                        .ToDictionary(binding => binding.Name, binding => binding.Data.ToObject());
+                    long bytesReceivedViaSharedMemory = 0;
+
+                    foreach (ParameterBinding binding in invokeResponse.OutputData)
+                    {
+                        switch (binding.RpcDataCase)
+                        {
+                            case ParameterBindingType.RpcSharedMemory:
+                                bytesReceivedViaSharedMemory += binding.RpcSharedMemory.Count;
+                                break;
+                            default:
+                                break;
+                        }
+                    }
+
+                    if (bytesReceivedViaSharedMemory > 0)
+                    {
+                        _workerChannelLogger.LogDebug("Received: {Size} bytes over shared memory for invocation Id: {Id}", bytesReceivedViaSharedMemory, invokeResponse.InvocationId);
```

Comment:
nit: Use StringBuilder to build log message in the format {BindingName}:{BytesReceived}